config/services.yml

```diff
@@ -983,3 +983,34 @@ https://cmr.uat.earthdata.nasa.gov:
         operations: ['reformat', 'concatenate']
         conditional:
           format: ['application/x-zarr']
+
+  - name: sds/HOSS-maskfill
+    # Provides polygon spatial subsetting for collections hosted in OPeNDAP
+    # Also provides variable and temporal subsetting through HOSS.
+    data_operation_version: '0.16.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/sds/HOSS
+    umm_s:
+      - S1245117629-EEDTEST
+    collections:
+      - id: C1222931739-GHRC_CLOUD
+    capabilities:
+      subsetting:
+        bbox: true
+        dimension: true
+        shape: true
+        variable: true
+      output_formats:
+        - application/netcdf # Incorrect mime-type, remove when no longer needed
+        - application/x-netcdf4
+    steps:
+      - image: !Env ${QUERY_CMR_IMAGE}
+      - image: !Env ${VAR_SUBSETTER_IMAGE}
+        operations: ['variableSubset', 'spatialSubset', 'dimensionSubset', 'shapefileSubset']
```

Comment:
Should the variable subsetter receive the shape information or should only the maskfill image receive the shape?